module/VuFind/src/VuFind/ILS/Connection.php

```diff
@@ -114,7 +121,9 @@ class Connection implements TranslatorAwareInterface, LoggerAwareInterface
      * Constructor
      *
      * @param \Zend\Config\Config              $config        Configuration
-     * representing the [Catalog] section of config.ini
+     *                                                        representing the
```

Comment:
It should not be necessary to align comments like this; with our code standards, they are allowed to wrap around to the beginning of the next line for improved readability. Is one of your tools getting overzealous here? :-)

Reply:
I fixed that :-)

Reply:
Thanks!